language/proto/generate.go

```diff
@@ -35,23 +35,11 @@ func (_ *protoLang) GenerateRules(args language.GenerateArgs) language.GenerateR
 		return language.GenerateResult{}
 	}
 
-	var regularProtoFiles []string
-	for _, name := range args.RegularFiles {
-		if strings.HasSuffix(name, ".proto") {
-			regularProtoFiles = append(regularProtoFiles, name)
-		}
-	}
-	var genProtoFiles []string
-	for _, name := range args.GenFiles {
-		if strings.HasSuffix(name, ".proto") {
-			genProtoFiles = append(args.GenFiles, name)
```

Comment:
@jayconrod this looks like it might be a bug. It should be 

```go
genProtoFiles = append(genProtoFiles, name) 
```

right?

Reply:
Yep, looks like a bug. Could you open a separate issue for this?

Reply:
Opened https://github.com/bazelbuild/bazel-gazelle/pull/455